From 4f093f79d33f8027874f6227df9533952123f4ea Mon Sep 17 00:00:00 2001
From: randomhydrosol <randomhydrosol@glassrom.org>
Date: Tue, 24 Jun 2025 10:07:22 +0530
Subject: [PATCH 104/105] Revert "kconfig: enable MSEAL_SYSTEM_MAPPINGS by
 default"

This breaks gvisor

This reverts commit 1d98adfc10517aec93e0182444814de47de5cbe6.
---
 security/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/security/Kconfig b/security/Kconfig
index e8714b6e99f96..38212ef51e11f 100644
--- a/security/Kconfig
+++ b/security/Kconfig
@@ -56,7 +56,6 @@ config MSEAL_SYSTEM_MAPPINGS
 	depends on 64BIT
 	depends on ARCH_SUPPORTS_MSEAL_SYSTEM_MAPPINGS
 	depends on !CHECKPOINT_RESTORE
-	default y
 	help
 	  Apply mseal on system mappings.
 	  The system mappings includes vdso, vvar, vvar_vclock,
-- 
2.50.1

